models/Schedule/Task.php

```diff
@@ -28,42 +28,42 @@ class Task extends Model\AbstractModel
     /**
      * @var int
      */
-    protected $id;
+    public $id;
```

Comment:
I think we should leave them `protected` and use `getObjectVars()` instead, in the controller.

Reply:
See https://github.com/pimcore/pimcore/pull/10320/commits/1f548009cd3c1b1357eaab98667609b3270b226e